package.json

```diff
@@ -201,7 +201,7 @@
     "style-loader": "~0.19.0",
     "uglifyjs-webpack-plugin": "^1.0.0",
     "uuid": "^3.0.1",
-    "webdriverio": "4.7.1",
+    "webdriverio": "^4.13.0",
```

Comment:
@NejcZdovc according to git blame, you set this to 4.7.1 for https://github.com/brave/browser-laptop/issues/10072. is that still a problem?

Reply:
looks like this is causing a bunch of deprecationWarnings but not errors (that weren't already there). https://travis-ci.org/brave/browser-laptop/jobs/394314325#L6366

can we set deprecationWarning false in this PR?